README.md

```diff
@@ -27,6 +28,38 @@ Rich IntelliSense (completions) for `Dockerfile` and `docker-compose.yml` files:
 
 ![IntelliSense for DockerFiles](images/intelliSense.gif)
 
+## Debugging (Preview)
+
+To debug a .NET Core web application running in a Linux Docker container, add a Docker .NET Core launch configuration:
+
+1. Switch to the debugging tab.
+1. Select `Add configuration...`
+1. Select `Docker: Launch .NET Core (Preview)`
+1. Set a breakpoint.
+1. Start debugging.
+
+> Mac OS X users: before debugging, add `/usr/local/share/dotnet/sdk/NuGetFallbackFolder` as a shared folder in your Docker preferences.
+
+Upon debugging, a Docker image will be built and a container run based on that image.  The container will have a volumes mapped to the locally-built application and the .NET Core debugger.  After the debugger is attached, the browser will be launched and navigated to the application's initial page.
+
+Most properties of the configuration are optional will be inferred from the project. If not, or if there are additional customizations to the Docker image build or container run process to be made, those can be added under the `dockerOptions` property of the configuration.
+
+```json
+{
+    "configurations": [
+        {
+            "name": "Docker: Launch .NET Core",
+            "type": "docker-netcoreapp",
+            "request": "launch",
+            "preLaunchTask": "build",
+            "dockerOptions": {
+                // Add customizations here.
```

Comment:
What are the options available?

Reply:
Added tables with descriptions of all of the options (to date).